README.md

```diff
@@ -408,6 +410,53 @@ var global_nav_config = {
 };
 ```
 
+###### Header.onDropdownOpen
+
+- Type: `String`, `Function`
+- Default: `null`
+
+Pass an `onDropdownOpen` property to enable callback when dropdown is open.
+Pass either a function, or the name of a function defined on `window`. E.g.:
+
+```js
+window.onDropdownOpenHandler = function () {
+  // Define your implementation here e.g.:
+  document.querySelector("body").classList.add("global-nav__dropdown--open");
+};
+
+var global_nav_config = {
+  header: {
+      onDropdownOpen: 'onDropdownOpenHandler',
+  },
+};
+```
+
+###### Header.onDropdownClose
+
+- Type: `String`, `Function`
+- Default: `null`
+
+Function parameters:
```

Comment:
Can you remove thus function parameters section here please as there are no args for these new hooks

Reply:
ahh yes missed those